sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/cpu/CpuMemoryReader.java

```diff
@@ -9,44 +9,56 @@
 
 import java.lang.management.ManagementFactory;
 
-public class CpuReader {
-    private final static Logger logger = LoggerFactory.getLogger(CpuReader.class);
+public class CpuMemoryReader {
+    private final static Logger logger = LoggerFactory.getLogger(CpuMemoryReader.class);
     private final com.sun.management.OperatingSystemMXBean operatingSystemMXBean;
 
-    public CpuReader() {
+    public CpuMemoryReader() {
         java.lang.management.OperatingSystemMXBean mxBean = null;
         try {
             mxBean =
                 ManagementFactory.getOperatingSystemMXBean();
         } catch (Throwable t) {
-            logger.error("failed to initialized CpuReader", t);
+            logger.error("failed to initialized CpuMemoryReader", t);
         }
 
         this.operatingSystemMXBean = tryGetAs(mxBean,
             com.sun.management.OperatingSystemMXBean.class);
     }
 
-    public double getSystemWideCpuUsage() {
+    public float getSystemWideCpuUsage() {
         try {
             if (operatingSystemMXBean != null) {
-                double val = operatingSystemMXBean.getSystemCpuLoad();
+                float val = (float) operatingSystemMXBean.getSystemCpuLoad();
```

Comment:
why can't we keep it `double`?

Reply:
We want to keep the size as low as possible, float give 6 to 7 place of decimal value , more than enough for us to measure cpu percentage.